libs/community/langchain_community/vectorstores/infinispanvs.py

```diff
@@ -382,19 +438,68 @@ class Infinispan:
     """
 
     def __init__(self, **kwargs: Any):
```

Comment:
nit: no change required here, but masking all the inputs in **kwargs** and unpacking in the init body instead of just doing it in the init function signature is fairly uncommon  as it introduces unintended bugs more often than not

Reply:
thanks, I unpacked the signature